src/utils/calculations/calculations.ts

```diff
@@ -0,0 +1,16 @@
+export const convertNumber = (num: number): string =>
```

Comment:
These function have names which tells me - they converts but to what ? I didn't notice this before. IMO you can rename convertNumber to convertNumberToKFormat - much longer but sometimes its better than generic name.

convertDate - is fine right now but probably later we change this